frontend/src/components/placementVisuals/AddPlacementContainer.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useCallback, useState } from "react"
+import { Alert, Button, Select, Tabs } from "antd"
+import { useAppDispatch, useAppSelector } from "../../hooks"
+import { selectCoordinatesByID } from "../../selectors"
+import { PLACED_STRING } from "./PlacementTab"
+import Modal from "antd/lib/modal/Modal"
+import SearchContainer from "../SearchContainer"
+import Input from "antd/lib/input/Input"
+import api from "../../utils/api"
+
+interface AddPlacementContainerProps {
+    onConfirm: (destinationContainer) => void
+}
+
+//component used to handle the creation of a new destination container
+const AddPlacementContainer = ({ onConfirm }: AddPlacementContainerProps) => {
+    const [isPopup, setIsPopup] = useState<boolean>(false)
+
+    //used to hold loaded container data
+    const [loadedContainer, setLoadedContainer] = useState<any>({})
+    const [selectedTab, setSelectedTab] = useState<string>('new')
+    const [error, setError] = useState<string | undefined>(undefined)
+    const [newContainer, setNewContainer] = useState<any>({})
+
+    const coordinates = useAppSelector(selectCoordinatesByID)
+
+    const dispatch = useAppDispatch()
+    //retrieves container on search
+    const handleContainerLoad = useCallback(async (loadedContainer) => {
+        const container = await dispatch(api.containers.get(loadedContainer))
+        const loadedSamples = await dispatch(api.samples.list({ id__in: container.data.samples.join(',') }))
```

Comment:
If the container doesn't contain any samples then the id__in list will be empty and the api will return the first 100 samples from ALL of the samples in the DB.
To reproduce, just create a new plate in Freezeman directly then try to select it as a Placement destination. The api will return 100 of 360,000 samples. When I do this, I end up getting an error saying "Unsupported container type" (???).

Reply:
The code bombs if one of the first 100 samples is a tube with an undefined coordinate. When you try to lookup the coordinate name it causes an "undefined" exception, and the container load fails, so later `handleConfirm` bombs because the selected container is undefined.

<img width="844" alt="Screen Shot 2024-01-24 at 3 57 36 PM" src="https://github.com/c3g/freezeman/assets/104078019/65517781-3add-43de-8889-a8a49775399d">